examples/next-fireworks/app/api/chat/route.ts

```diff
@@ -0,0 +1,28 @@
+import OpenAI from 'openai'
+import { OpenAIStream, StreamingTextResponse } from 'ai'
+
+// Create an OpenAI API client (that's edge friendly!)
+// but configure it to point to fireworks.ai
+const fireworks = new OpenAI({
+  apiKey: process.env.OPENAI_API_KEY || '',
+  baseURL: process.env.OPENAI_API_BASE // "https://api.fireworks.ai/inference/v1"
```

Comment:
just curious: why the endpoint is a comment and not a default value for the env var?

Reply:
Just reminding folks in the comment

Reply:
I meant you could also put `process.env.OPENAI_API_BASE || "https://api.fireworks.ai/inference/v1"` in case they omit it. But doesn't really matter. Thanks!

Reply:
oh i see. yeah we can do that. Good idea!